docs/getting-started/development-environment.rst

```diff
@@ -178,7 +178,105 @@ images are having problems building.
 Linux
 ~~~~~
 
-Want to write it? Contact us on the chat!
+Useful packages to install:
```

Comment:
I would then create a specific subsection `Ubuntu v....` because this is all highly dependent of the Linux dist. :)